bin/jenkins-dist-publish.sh

```diff
@@ -25,14 +25,14 @@ function rename_dist {
     SPARK_DIST_DIR="spark-${SPARK_VERSION}-bin-${HADOOP_VERSION}"
     SPARK_DIST="${SPARK_DIST_DIR}.tgz"
 
-    mkdir -p "${SPARK_DIST_DIR}"
+    mkdir -p $SPARK_DIST_DIR
```

Comment:
Why did we remove the quoting in this function?

Reply:
I'll revert those. I didn't think they were important but they are.